caddyhttp/fastcgi/fastcgi.go

```diff
@@ -35,6 +41,9 @@ type Handler struct {
 func (h Handler) ServeHTTP(w http.ResponseWriter, r *http.Request) (int, error) {
 	for _, rule := range h.Rules {
 
+		//TODO: add an option in Caddyfile and pass it down to here
+		usePersistentFcgiConnections := runtime.GOOS == "windows"
```

Comment:
Is there a technical reason this is limited to Windows (for now)? Is there any performance gain on Linux too? (maybe under load?)